torch/fx/interpreter.py

```diff
@@ -189,6 +190,8 @@ def run_node(self, n : Node) -> Any:
             Any: The result of executing ``n``
         """
         with self._set_current_node(n):
+            # Sets the seq_nr for the fwd pass
+            fx_traceback.set_seq_nr(get_sequence_nr())
```

Comment:
Still not sure why this is needed. It doesn't look like we we check what the value of this is later. And if during the forward pass it would be overwritten by new_seq, and if during the backward pass it would be overwritten by whatever the hooks set.

Reply:
Removed it.